apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -473,6 +475,12 @@ class TopInstructions extends Component {
   handleHelpTabClick = () => {
     this.handleTabClick(TabType.RESOURCES);
     this.recordEvent('click-help-and-tips-tab');
```

Comment:
side note: does anyone look at this data? after keeping it around for a bit to maybe confirm that uli matches what's logged via firehose, we can probably delete this? or should at least have a discussion about it